src/wallet/wallet2.cpp

```diff
@@ -6197,7 +6262,13 @@ void wallet2::get_outs(std::vector<std::vector<tools::wallet2::get_outs_entry>>
 
           uint64_t i;
           const char *type = "";
-          if (num_found - 1 < recent_outputs_count) // -1 to account for the real one we seeded with
+          if (amount == 0 && has_rct_distribution)
+          {
+            // gamma distribution
+            do i = pick_gamma(); while (i >= num_outs);
```

Comment:
This means the pre/post-fork segregation is deactivated for this scheme. Is this intentional?